cmd/utils/utils.go

```diff
@@ -95,3 +95,10 @@ func ReadFile(filePath string) (byteValue []byte, err error) {
 	}
 	return byteValue, err
 }
+
+func TestFolder(folder string) bool {
```

Comment:
May I suggest naming this `IsFolder` instead? The current name would tend to indicate that this has something to do with unit-tests rather than what it is actually doing.

Or even better, not refactoring it into a separate function?

The code is simple enough that refactoring it does not not change the number of lines required in the callers. Hence it only adds complexity to understand what is going on. A reader now needs to look at a separate file instead of being able to read the code linearly.

Reply:
Thank you for reviewing. I have addressed it now.